docs/requirements.txt

```diff
@@ -4,4 +4,6 @@ myst-parser==0.13.6
 myst-nb==0.12.3
 sphinx-copybutton==0.3.1
 sphinx-autodoc-typehints
+sphinxcontrib-openapi
+uvicorn
```

Comment:
Out of curiosity, why does this require uvicorn?

Reply:
Good catch. Originally I thought you had to run the server, but I found a way to avoid that.